src/api/sirius_api.cpp

```diff
@@ -6193,11 +6202,15 @@ sirius_linear_solver(void* const* handler__, double const* vkq__, int const* num
 
                 // Identity_preconditioner preconditioner{static_cast<size_t>(nbnd_occ)};
 
+                auto tol = get_value(tol__, 1e-13);
+
                 auto result = sirius::cg::multi_cg(linear_operator, preconditioner, X_wrap, B_wrap, U_wrap,
                                                    C_wrap, // state vectors
                                                    100,    // iters
```

Comment:
probably we have to remove the hard-coded max number of iterations as well, but that can be done in another PR